modules/ROOT/pages/dataweave-memory-management.adoc

```diff
@@ -4,20 +4,20 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, esb, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping, streaming, weaveoutputhandler
 
-When processing large files through DataWeave in Mule, there are a few things you can set up to fine-tune how much memory will be used up and when.
-//LINK TO DW 1.0 LANDING PAGE IN MULE 3.9 DOCS:
-include::partial$dataweave1-links.adoc[tag=dataweave1LandingPage]
+When processing large files through DataWeave in Mule runtime engine, there are a few things you can set up to fine-tune how much memory will be used and when.
```

Comment:
See suggestion above. Otherwise:
```suggestion
To fine-tune processing large files through DataWeave in Mule runtime engine, you can set controls for how much memory is used, and when.
```